modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -30,6 +30,23 @@
 
 The following premium plugin updates were released alongside {productname} 6.5.
 
+=== Enhanced Media Embed 3.1.2
+
+The {productname} 6.5 release includes an accompanying release of the **Enhanced Media Embed** premium plugin.
+
+**Enhanced Media Embed** 3.1.2 includes the following fixes, additions, and improvements.
+
+=== Allow a media embed element to be correctly resized when using the `media` plugin dialog by converting the responsive media embed to a standalone iframe.
+//#TINY-8714
+
+In previous versions of {productname}, the Mediaembed plugin and the media plugin encountered several issues when used together. The Mediaembed plugin would wrap `<iframes>` with a responsive wrapper that relies on large padding for the height. However, when content created with the Mediaembed plugin is inserted using the media plugin dialog, which targets `"ephox-embed-iri"` divs, the width and height are mapped to the wrapper's `max-width` and `max-height` properties. Unfortunately, the height is primarily determined by padding, rendering the `max-height` property ineffective.
```

Comment:
```suggestion
In previous versions of {productname}, the Enhanced Media Embed and the media plugin encountered several issues when used together. The Mediaembed plugin would wrap `<iframes>` with a responsive wrapper that relies on large padding for the height. However, when content created with the Mediaembed plugin is inserted using the media plugin dialog, which targets `"ephox-embed-iri"` divs, the width and height are mapped to the wrapper's `max-width` and `max-height` properties. Unfortunately, the height is primarily determined by padding, rendering the `max-height` property ineffective.
```

Should this and the rest of the places be called as Enhanced Media Embed plugin?

Reply:
@shanmen-tiny I agree, ill updated all of them to reflect `Enhanced Media Embed plugin`